modules/cert-manager-issuer-types.adoc

```diff
@@ -4,13 +4,33 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="cert-manager-issuer-types_{context}"]
-= Supported issuer types
+= {cert-manager-operator} Issuer Providers
 
-The {cert-manager-operator} supports the following issuer types:
+The cert-manager operator for Red Hat OpenShift has been tested with the following issuer types:
 
 * Automated Certificate Management Environment (ACME)
-* Certificate authority (CA)
 * Self-signed
+* link:https://cloud.google.com/security/products/certificate-authority-service[Google cloud Certificate Authority Service] (Google CAS)
```

Comment:
Please move `Google CAS` to the botton. Becasue the NCM and Google CAS are external issuers. We doesn't and won't fully test external issuers.

It'd be better to list them (external issuers) after in-tree issuer types (ACME, CA, Self-signed, Vault, Venafi).